livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -422,6 +423,8 @@ private String callRestAPI(String targetURL, String method, String jsonData)
         if (cause.getResponseBodyAsString().matches(SESSION_NOT_FOUND_PATTERN)) {
           throw new SessionNotFoundException(cause.getResponseBodyAsString());
         }
+        throw new LivyException(cause.getResponseBodyAsString() + "\n"
+            + ExceptionUtils.getFullStackTrace(ExceptionUtils.getRootCause(e)));
```

Comment:
Yes, this is the default behavior of zeppelin interpreter. See
https://github.com/apache/zeppelin/blob/master/zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/Job.java#L191